src/config.py

```diff
@@ -138,6 +138,8 @@ def flip_red(pos: Translation2d) -> Translation2d:
     (1.636584300, 1.5),
     (1.906579590, 0),
 ])
+
+pivot_backlash_compensator = 0
```

Comment:
Why is it before the `# fmt: on`? Also I feel like there should be a comment here explaining what this constant actually does.